checkbox.js

```diff
@@ -0,0 +1,125 @@
+/*jshint esversion: 6*/
+/*global document*/
+/*global alert*/
+
+class Data {
+    static getCheckboxList() {
+        return ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday', 'None'];
+      } 
+  }
+  
+  class CheckboxOperations {
+    
+    createCheckboxList() {
+      let ul = this.createUlElementForCheckbox();
+      document.body.appendChild(ul);
+        this.createAndAppendCheckboxListTo(ul);
+    }
+  
+    createAndAppendCheckboxListTo(ul) {
+        let checkboxIds = Data.getCheckboxList();
+        checkboxIds.forEach((checkboxId) =>{
+        let label = this.createLabelElement(checkboxId);
+              let checkbox = this.createCheckbox(checkboxId);
+        
+        label.appendChild(checkbox);
+        label.appendChild(document.createTextNode(checkboxId));
+        
+        ul.appendChild(this.createLiElementAndAppend(label));
+  
+      });
+      }
+    createUlElementForCheckbox() {
+        let ul = document.createElement('ul');
+      ul.style.listStyle = 'none';
```

Comment:
Do something like: DEFAULT_LIST_STYLE will be const obj having `{listStyle: "none", ..other attributes etc}`
`ul.style = DEFAULT_LIST_STYLE`.